src/index.html

```diff
@@ -112,7 +116,8 @@ <h4 class="item-name">Name</h4>
                         <div class="moovie-item-image-description">
                             <h4 class="item-name">Name</h4>
                             <div class="moovie-item-likes">
-                                <p class="heart"><i class="fa-regular fa-heart" id="like"></i></p>
+                                <p class="heart"><i class="fa-regular fa-heart" class="like"></i></p>
+                                <input type="hidden" name="idmoovie" value="" class="moovieId">
                                 <p class="likes-number">25 Likes</p>
                             </div>
                         </div>
```

Comment:
Optional Changes:
Although it is not a requirement, I would suggest that you set the counter likes to initially begin from zero so that the number changes the page loads.♻. this would look nice.🙂

Reply:
thanks I will do